WordPress/gutenberg
# Query Loop block: Add 8 new patterns

Follow up to https://github.com/WordPress/gutenberg/pull/30763, closes https://github.com/WordPress/gutenberg/issues/44140

It's been a while since we've updated the bundled patterns for the Query Loop block. I wanted to share a few new options, especially some more text-based options as discussed in https://github.com/WordPress/gutenberg/issues/44140

This PR adds 8 new patterns for the Query Loop block. If we move forward with adding some of these new patterns, we may also want to go through and remove some of the existing image-based ones. Here are names and screenshots of the patterns added:

**Post feed**

<img src="https://user-images.githubusercontent.com/77359364/191310565-35d64e36-d0fb-4ff8-b339-6f135b643396.png">

**Post list cards**

<img src="https://user-images.githubusercontent.com/77359364/190220409-f3ca86f4-fd04-41f1-85e8-9d1f9e844482.png">

**Post table**

<img src="https://user-images.githubusercontent.com/77359364/190220454-e319b106-aa00-40bf-9532-548fe1d30e6e.png">

**Two column text list**

<img src="https://user-images.githubusercontent.com/77359364/190220501-ef28843e-d85f-4566-833e-1d61b3155e91.png">

**Two column with tags**

<img src="https://user-images.githubusercontent.com/77359364/190220551-18f0d5f9-61a0-4737-aaa3-7c5c6cbcb587.png">

**Colorful full width posts**

<img src="https://user-images.githubusercontent.com/77359364/190220584-2fe4b5ec-14b7-4be9-974e-dd35f674cbc0.png">

**Featured post with post list**

<img src="https://user-images.githubusercontent.com/77359364/190220620-1070de29-5d21-4f1b-96ce-786e1fd7eaca.png">

**More posts**

<img src="https://user-images.githubusercontent.com/77359364/190220666-9fd8c314-6d5b-42f0-8351-1b92f697c283.png">

**Post and date list**
(Adapted from [Beaumont theme](https://wordpress.org/themes/beaumont/))

<img src="https://user-images.githubusercontent.com/77359364/193334788-234ddfaf-ad7a-488f-a437-aa711fb0719d.png">

Note: the last three patterns are combinations of blocks that contain a Query Loop block within.

## Testing Instructions
1. Open the site editor
2. Select the Query Loop block and select "Replace" from the toolbar to open the pattern explore modal
3. Check to see whether the new patterns appear in the modal
4. Insert the patterns and make sure they look ok in the template

## Requested feedback
Let's figure out which of these patterns we'd like to add and which of the existing Query Loop patterns should be removed. cc [@WordPress/gutenberg-design](https://github.com/orgs/WordPress/teams/gutenberg-design)
